docs/modules/integrations/pages/webhooks/create-and-manage-webhooks.adoc

```diff
@@ -0,0 +1,128 @@
+
+Select the **Webhook Settings** tab.
+
+image:integrations:w2.png[]
+
+Select **Create Webhook**.
+
+image:integrations:w3.png[]
+
+Complete the required information as described below:
+
+[options="header"]
+|=======================
+|Parameter|Required|Format|Example|Description
+|Webhook Name|Yes|case-sensitive, 50 characters maximum|Device offline|Name of the webhook; it must be unique to the org Example: Webhook for online status
+|URL|Yes|512 characters maximum, valid URL syntax|https://webhook.site/49e1592a-6736-4b8b-bd58-c97ab|The URL where the callback should be sent
+|Description|No|512 characters maximum|Webhook for offline devices|Description of the webhook
+|Event|Yes (at least 1 event enabled)|On/off toggle| |The type of event to trigger the callback
+|=======================
+
+image:integrations:w4.png[]
+
+Choose one or more event triggers by enabling the toggle of each event. Then, select *Save*.
+
+image:integrations:w5.png[]
+
+== Execute a webhook
+
+When a webhook is created, it tracks the status of the Private and Local devices in an org. It sends HTTP callbacks to the specified URL when a condition is met for those devices.
+
+The body of the callback will be similar to the below:
+
+[source,JavaScript]
+----
+{
+  "creationTime": "2023-01-06T04:52:36.963Z",
+  "eventType": "DEVICE",
+  "eventActivity": "device.state.utilizing",
+  "eventDescription": "Utilizing",
+  "deviceUdid": "ce0917198075440c057e",
+  "deviceName": "Galaxy Note 8",
+  "orgId": 2,
+  "webhookId": 129,
+  "webhookName": "Webhook for All events"
+}
+----
+
+See below for details of the parameters in the example above:
+
+
+[options="header"]
+|=======================
+|Parameter|Description
+|creationTime|The date and time the callback is sent
+|eventType|The type of event
+|eventActivity|The triggering event
+
+For offline and online events, it appears with the format: device.status.<event>
+
+For unplugged and utilizing events, it appears with the format: device.state.<event>
+|eventDescription|The description of the webhook triggering event
+|deviceUdid|The UDID of the device
+|deviceName|The name of the device
+|orgId|The unique ID of the organization
+|webhookId|The unique ID of the webhook
+|webhookName|The webhook name
+|=======================
+
+== Delete a webhook
+
+If you no longer want to receive a callback, delete the webhook.
+
+Navigate to *Webhook Settings*, select a webhook, and choose *Delete*.
+
+image:integrations:w6.png[]
+
+Confirm the deletion of the webhook by selecting *Delete* again in the modal:
+
+image:integrations:w7.png[]
+
+== Edit a webhook
+
+Go to *Webhook Settings*, select a webhook and choose *Edit*.
+
+image:integrations:w8.png[]
+
+Make changes and select *Save*.
+
+image:integrations:w9.png[]
+
+== Verify a webhook
+
+To quickly verify a webhook created Kobiton, use an online webhook testing site such as link:https://webhook.site/[webhook.site] or build your own API to receive the webhook.

```

Comment:
Change beginning to: To quickly verify a Kobiton webhook, use"

Reply:
Changed

Reply:
Verified